src/Entity/ProcurationRequest.php

```diff
@@ -344,14 +351,48 @@ public function process(ProcurationProxy $procurationProxy = null, Adherent $pro
         $this->processedAt = new \DateTimeImmutable();
 
         if ($procurationProxy) {
-            $procurationProxy->setFoundRequest($this);
+            $proxiesUsedCount = $procurationProxy->getFoundRequests()->count();
```

Comment:
I'm pretty sure those two logics (from process and unprocess) could be improved, open to discussions :)